lrtable/src/lib/statetable.rs

```diff
@@ -303,10 +303,8 @@ where
     }
 
     fn decode(bits: u32) -> Action<StorageT> {
-        //           Action   State/Prod
-        let mask = 0b00000000_11111111;
-        let action = bits >> 8;
-        let reference = bits & mask;
+        let action = bits & 0b11;
+        let reference = bits >> 2;
```

Comment:
Let's call this `val` or `v` or something. `reference` is very long and misleading.